modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/experience/components/ExperienceToolbarSection.test.js

```diff
@@ -697,4 +700,72 @@ describe('ExperienceToolbarSection', () => {
 			})
 		);
 	});
+
+	it('calls the backend to duplicate an experience', async () => {
```

Comment:
I miss to check if in the list after the `serviceFetch` there is an experience name called "Experience #1 (Copy)" 🤔

Reply:
I totally agree, thanks @dgarciasarai

Reply:
I don't know if we can check it because we are mocking the fetch petition...